### ISSUE TITLE
An introduction to DOM Observers

### ISSUE THREAD

**Issue by stowball** [newwwie/newcastlejs-meetup#3]
## Title
An introduction to DOM Observers

## About
MutationObserver, IntersectionObserver and ResizeObserver are awesome new DOM APIs that can perform magic. Let me show you how.

## Duration
30 mins

## Target Audience
Intermediate – Advanced

## Date
Whenever

**Comment by neozenith** [comment]
That’s sound cool. We have a few tentative speakers queued up so how about Oct or Nov?

**Comment by stowball** [comment]
Yeah, let others have a chance before me haha. Either should be fine at this point

**Comment by neozenith** [comment]
oops closed the wrong issue.

**Comment by neozenith** [comment]
@stowball would you still be interested in presenting this sometime this year?

**Comment by stowball** [comment]
@neozenith Yeah, I'm happy to. But it won't be until after March

**Comment by neozenith** [comment]
Awesome! Mostly checking in case you got bored of the topic and wanted to move on. I'll circle back later in the year.

**Comment by jnowland** [comment]
Did you do this during your epic 2018 speaker run?

We have 11 September 2019  free but who knows you may be more focused on utility libs now days. :)
